cloudconfig/userdatacfg_unix.go

```diff
@@ -86,6 +87,12 @@ rm -rf /var/lib/juju/raft/*
 
 echo "removing /var/run/juju/*"
 rm -rf /var/run/juju/*
+
+has_juju_db_snap=$(snap info juju-db | grep installed)
+if [ ! -z "$has_juju_db_snap" ]; then
+        echo "removing juju-db snap and any persisted database data"
+	snap remove --purge juju-db
```

Comment:
Tabs vs Spaces: Fight!

So line 93 is spaces, 94 is a tab...

Reply:
Doh!